vellum/workflows/triage_message/nodes/store_outbox_message_node.py

```diff
@@ -21,9 +23,10 @@ class StoreOutboxMessageNode(BaseNode):
 
     class Outputs(BaseNode.Outputs):
         summary: str
+        message_url = LazyReference(lambda: f"/admin/inboxes/{ReadMessageNode.Outputs.message['inbox_id']}/messages/{ReadMessageNode.Outputs.message['message_id']}")
```

Comment:
We shouldn't need LazyReference here either

Reply:
Actually vellum doesn't support string con at like this yet. Specify this as just a str output and concatenate the string within the run method